packages/webcomponents/src/components/payments-list/get-payments.ts

```diff
@@ -0,0 +1,29 @@
+import { Payment } from '../../api';
+
+export const makeGetPayments =
```

Comment:
can this function be renamed to `getPayments` ? 

I'm finding `makeGetPayments` a little confusing to read

Reply:
Thanks for the feedback! makeGetPayments is actually a factory function, hence the 'make' prefix—it sets up a custom function with specific settings. I understand the confusion, though. What do you think about `createPaymentFetcher` as an alternative? It might strike a better balance.

Reply:
Ah I see - thank you for explaining @TiagoFuelber ! 

I think I like `createPaymentFetcher` a little better, but in that case I will let you decide since its a factory function. I can live with either lol.